src/Components/CreateEvent/CreateEvent.css

```diff
@@ -0,0 +1,47 @@
+body {
+    background-color: var(--secondary-color-white);
+}
+
+.line-break {
+    background-color: var(--secondary-color-white);
+    width: 50.25vw;
+    height: .25vh;
+}
+
+.card {
+    height: var(--section-height);
+    width: var(--section-width);
```

Comment:
Loving the use of these variables across the app - it'll make everything so much easier